quantum/led_matrix/led_matrix.c

```diff
@@ -459,7 +459,7 @@ void led_matrix_init(void) {
 
 void led_matrix_set_suspend_state(bool state) {
 #ifdef LED_DISABLE_WHEN_USB_SUSPENDED
-    if (state) {
+    if (state && is_keyboard_master()) {
```

Comment:
Hm, why does LED Matrix need this but RGB Matrix not? Could it be something in the ISSI driver itself?

Reply:
Not sure. Looking at the code of `led_matrix_set_value_all`, it seems it _shouldn't_ make any difference, but it does. 😕

Reply:
I've done some more testing, and this is really weird.

If I skip the special handling of `LED_MATRIX_SPLIT` in either `led_matrix_set_value_all` or `led_matrix_set_value`, everything works fine, for Ergodox Infinity at least (since its LED pinouts are the same on each half, just mirrored positions). The weird part of this is that the first and second halves of `g_is31_leds` are identical, so the only real difference _should_ be that it's writing the led values twice. 😵 

Note that the serial comms die after suspend regardless of which side is the master (without any change).